### PR TITLE
Add time package

### PR DESCRIPTION
**Pull request by dc7303**
### What this PR does / why we need it?
Add the time package with actor_id and ticket.

### What are the relevant tickets?

Fixes #

### Checklist
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

**Review by blurfx — APPROVED** [review]
LGTM 👍

File: src/document/time/actor_id.rs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        ActorID { bytes }
+    }
+
+    /// from_hex returns the ActorID represented by the hexadecimal string str.
+    pub fn from_hex(hex_str: &str) -> Result<ActorID, FromHexError> {
+        if hex_str == "" {
+            return Err(FromHexError::InvalidStringLength);
+        }
+
+        let bytes = <[u8; 12]>::from_hex(hex_str)?;
```
Comment (hackerwins) [inline src/document/time/actor_id.rs:24]: I'm curious about how to handle errors in Rust. When inputting an incorrect hex string, how is the error handled?

**Review by hackerwins — APPROVED** [review]
Thanks for the work. 👍 
I left a simple question.

File: src/document/time/actor_id.rs (hunk 1 above)
Comment (dc7303) [reply src/document/time/actor_id.rs:24]: Rust can handle error handling with `match`, but there is a [? operator](http://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/reference/expressions/operator-expr.html#the-question-mark-operator) that can reduce repetitive code.  

Simply put, the code above is equivalent to the code below.
```
        let bytes = match <[u8; 12]>::from_hex(hex_str) {
            Ok(bytes) => bytes,
            Err(err) => return Err(err),
        };
```

And if an incorrect input value is entered, the error defined in [hex::FromHexError](https://docs.rs/hex/0.3.1/hex/enum.FromHexError.html) is returned.
If you need mapping like in go, there is also [this way](https://doc.rust-lang.org/rust-by-example/error/multiple_error_types/wrap_error.html).

>TMI: Rust returns Result using variants called Ok and Err. And the value that enters Err can be of various types such as struct, enum, string, etc.

**Pull request merged**